.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "lldb",
+      "request": "launch",
+      "name": "Debug executable 'rslint'",
+      "cargo": {
+        "args": [
+          "build",
+          "--bin=rslint",
+          "--package=rslint_cli"
+        ],
+        "filter": {
+          "name": "rslint",
+          "kind": "bin"
+        }
+      },
+      "args": ["-f", "${workspaceFolder}/tests/main.ts"],
+      "cwd": "${workspaceFolder}"
+    },
```

Comment:
I think it's still helpful to add `"sourceLanguages": ["rust"]`. Not sure what it does but it makes sense

Reply:
Thanks for the suggestion, I have updated.